subhub/tests/unit/test_payment_calls.py

```diff
@@ -268,8 +274,9 @@ def test_delete_customer(app, create_subscription_for_processing):
     # (subscription, code) = create_subscription_for_processing
     (message, code) = payments.delete_customer("process_test")
     assert message["message"] == "Customer deleted successfully"
-    deleted_message, code = payments.subscription_status("process_test")
-    assert deleted_message["message"] == "Customer does not exist."
+    (deleted_message, code) = payments.subscription_status("process_test")
+    print(f"deleted message {deleted_message}")
```

Comment:
is this a print that was accidentally checked in? if not why not use the logger?

Reply:
removed and yes.
(when running pytest sometimes I want to print out a statement when debugging, logger does not get printed to the console when there is an error)